StitchApp/RegressionTestAppFunctions/compareChildJobs/source.js

```diff
@@ -0,0 +1,63 @@
+exports = function(commitHash, testJobs, prodJobs){
+  //retrieve child jobs from regression database 
+  //somehow make a list
+  const repos = context.functions.execute('getReposApprovedForTesting');
+  repos.forEach((repo) => {
+    const testJob = testJobs[repo];
+    const prodJob = prodJobs[repo];
+    let testTime;
+    let prodTime;
+    let testLink;
+    let prodLink;
+    let testLinkCode;
+    let prodLinkCode;
+    console.log(testJob.numFailures);
+    console.log(prodJob.numFailures);
+    
+    //compare status
+    const testStatus = testJob.status;
+    const prodStatus = prodJob.status;
+    const testLogs = testJob.logs; 
+    const prodLogs = prodJob.logs;
+    //compare time it took in seconds, only get an endTime if job completed 
+    //also compare results of curl-ing the link in Summary, which is only generated if job completes
+    if (testStatus === "completed"){
+      testTime = (testJob.endTime - testJob.startTime)/1000;
+      //lastMessage.indexOf('Summary')
+      testLink = testLogs[testLogs.length - 1].split('Hosted at:')[0];
```

Comment:
We need to adjust our measure of fail and success and rely solely on status (this will be possible once we parameterize the parser to emit a fail status of diagnostics). That way we can leverage each component of snooty to do its job in reporting pass and fail state rather than coding brittle measures. For now, let's rely on status to at least capture catastrophic failures and once the diagnostics can be set to influence the status, we'll have better coverage.